sphinxmixcrypto/client.py

```diff
@@ -35,6 +35,24 @@ def destination_encode(dest):
 
 
 def create_header(params, route, pki, dest, message_id, rand_reader):
+    """
+    Create a sphinx header, used to construct forward messages and reply blocks.
+
+    :param params: An instance of SphinxParams.
+
+    :param route: A list of 16 byte mix node IDs.
+
+    :param pki: An implementation of IMixPKI.
+
+    :param dest: A "prefix free encoded" destination type or client ID.
+
+    :param message_id: Message identifier.
+
+    :param rand_reader: Source of entropy, an implementation of IReader.
+
+    :returns: a 2-tuple, a SphinxHeader and a list of shared secrets
+    for each hop in the route.
```

Comment:
32 bytes... curve25519 diffie hellman key agreement derived. yeah i should make a type for this too.